qualtricssurvey/qualtricssurvey.py

```diff
@@ -39,7 +39,7 @@ class QualtricsSurvey(StudioEditableXBlockMixin, XBlock):
     )
     your_university = String(
         display_name=_('Your University:'),
-        default=_('stanford'),
+        default=_('stanforduniversity'),
```

Comment:
I think we actually _don't_ want this marked for translation, right? Isn't this like an account name?